lib/Doctrine/DBAL/Connections/MasterSlaveConnection.php

```diff
@@ -167,12 +167,12 @@ public function connect($connectionName = null)
             $this->connections['slave'] = $this->_conn = $this->connectTo($connectionName);
         }
 
-        if ($this->_eventManager->hasListeners(Events::postConnect)) {
-            $eventArgs = new ConnectionEventArgs($this);
-            $this->_eventManager->dispatchEvent(Events::postConnect, $eventArgs);
+        if (! $this->_eventManager->hasListeners(Events::postConnect)) {
+            return;
         }
 
-        return true;
+        $eventArgs = new ConnectionEventArgs($this);
```

Comment:
Variable redundant

Reply:
It was like this prior to the change and only got changed because of `EarlyExitNotUsed` which often makes more harm than good if enforced (e.g. https://github.com/slevomat/coding-standard/issues/371).